galaxy/templates/priorityclass-job.yaml

```
@@ -5,8 +5,8 @@ metadata:
   name: {{ include "galaxy.pod-priority-class" . }}
   labels:
     {{- include "galaxy.labels" . | nindent 4 }}
-value: -1000
+value: {{ default "-1000"  .Values.jobs.priorityClass.value -}} 
```

This needs a bit of testing, but the last line chomp might break things?
```suggestion
value: {{ default "-1000"  .Values.jobs.priorityClass.value }} 
```

---

Yeah, the next line is pulled up to line 8 now. Removing the Minus/Dash.